test/e2e/framework/pod/create.go

```diff
@@ -167,11 +166,12 @@ func MakeSecPod(podConfig *Config) (*v1.Pod, error) {
 	if podConfig.NS == "" {
 		return nil, fmt.Errorf("Cannot create pod with empty namespace")
 	}
-	if len(podConfig.Command) == 0 {
+	if len(podConfig.Command) == 0 && !NodeOSDistroIs("windows") {
 		podConfig.Command = "trap exit TERM; while true; do sleep 1; done"
```

Comment:
with this other change:

```
	if command == "" {
		return commands
	}
```

is the pod still running in an endless loop?

Reply:
I think so, without running any commands, agnhost container will keep running.

Reply:
I also feel for most cases including linux, this command set up is not necessary. The test container most already does it.

Reply:
got it, thanks for sharing this, there was another place in the tests where we I saw sleep, I'm going to change that later